### ISSUE TITLE
Tinymce iframes are not associated to their labels

### ISSUE THREAD

**Issue by briri** [DMPRoadmap/roadmap#1449]
Screen readers currently read 'frame clickable clickable clickable clickable' when the user navigates to the Tinymce editors. 

I believe this is due to the fact that Rails form helpers that we use to create our labels and input fields is associating the label with the underlying (hidden) input field that Tinymce stores the user content in when the page is submitted and not the iframe.

The recommendation was to add a title to the iframe, but this would create a tooltip that repeats the label or question (for the edit plan page). I think we should try updating the label's 'for' attribute to properly connect it to the iframe first. If that fails we can add the 'title' attribute

This feedback was provided from the UCOP electronic accessibility review. It was defined as having 'High' priority.

**Comment by mariapraetzellis** [comment]
@briri will work on this one. We'll push it out for the next release since it will take a while to develop.

**Comment by mariapraetzellis** [comment]
Brian working on this today.

**Comment by mariapraetzellis** [comment]
Can't UAT so going to push this on through.
